Scripts/Core/VRView.cs

```diff
@@ -333,6 +346,7 @@ void UpdateHMDStatus()
 
 		static bool GetIsUserPresent()
 		{
+#if UNITY_2017_2_OR_NEWER
```

Comment:
Kind of want to do a bigger define wrapper here as well but I see things aren't quite as adjacent/easily grabbed together.  Thoughts?

Reply:
Well, there isn't a need for backwards compatibility moving forward, so why should anything compile from EditorVR when the version of Unity is not 2017.2 or newer?

Reply:
The danger with blocking out *all* of the EVR code behind this kind of fence is that a user can hose their scriptable objects (like their editing contexts) because their meta files and references are just meaningless now.  This has happened to us more than once already during this upgrade.

Reply:
It also has to return *something* or it will cause compile errors.

Reply:
I also didn't want to have to refactor uses of methods, so as much as possible I just focused on implementations.